cmd/root/auth_test.go

```diff
@@ -15,7 +15,8 @@ import (
 )
 
 func TestEmptyHttpRequest(t *testing.T) {
-	ctx, _ := context.WithCancel(context.Background())
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
```

Comment:
why do you need to defer call it?

Reply:
Not sure what you suggest? To cancel it right away? This would change this test, because previously it worked on un-cancelled context.

Reply:
Ah, I see, linter forces you to use the `cancel` fn here, got it